File: 18/18/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace _18
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }
          private void label2_Click(object sender, EventArgs e)
                {
                }
        private void button1_Click(object sender, EventArgs e)
        {
            
                int cont = 4;
            for (int i = 0; i < 3; i++)
            {
                int a = int.Parse(Microsoft.VisualBasic.Interaction.InputBox("Digite un numero", "Adivinar contraseña"));
                if (a == cont)
                {
                    MessageBox.Show("Acceso permitido");
                   
                    i = 3;
                }
                else
                {
                    if (i == 2)
                    {
                        MessageBox.Show("Clave bloqueada");
                       
                    }
                    else
                    {
                        MessageBox.Show("Acceso denegado");
                    }
                }
            }
            
           
        }

        private void button2_Click(object sender, EventArgs e)
        {
            this.Close();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
          
        }
    }
}

File: 19/19/Form1.vb
Public Class Form1
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click


        Dim a = CInt(TextBox1.Text)

        If (a = 4) Then
            Label2.Text = "Acceso permitido"
            TextBox1.Enabled = False
        Else
            Label2.Text = "Acceso denegado"
            TextBox1.Text = " "
        End If

    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        Close()
    End Sub
End Class
